jbmc/src/java_bytecode/ci_lazy_methods_needed.cpp

```diff
@@ -42,6 +61,14 @@ bool ci_lazy_methods_neededt::add_needed_class(
   if(symbol_table.symbols.count(cprover_validate))
     add_needed_method(cprover_validate);
 
+  // Special case for enums. We may want to generalise this, see TG-4689
+  // and the comment in java_object_factoryt::gen_nondet_pointer_init.
+  namespacet ns{symbol_table};
```

Comment:
Why {}?

Reply:
Copy-pasted from another function in this file, but `()` is probably better so I changed it to that, thanks.